apidoc2swagger/apidoc2swagger.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ *
```

Comment:
Yes i agree with the MIT license. But since you are the repository owner you have to add that license i think.